saffron/src/blob.rs

```diff
@@ -161,4 +206,47 @@ mod tests {
             prop_assert_eq!(user_commitments, blob.commitments);
           }
         }
+
+    fn random_perturbation(threshold: f64, data: &[u8]) -> Vec<u8> {
+        let mut rng = rand::thread_rng();
+        data.iter()
```

Comment:
`par_iter` also here I guess.

Reply:
see [this comment](https://github.com/o1-labs/proof-systems/pull/3004#discussion_r1945469219)

Reply:
See [this comment](https://github.com/o1-labs/proof-systems/pull/3004#discussion_r1948110106)